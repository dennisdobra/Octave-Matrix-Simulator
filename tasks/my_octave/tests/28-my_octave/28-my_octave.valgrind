==62561== Memcheck, a memory error detector
==62561== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==62561== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==62561== Command: /home/dominic/Documents/Facultate_sem_3/PCLP1_asist/check_my_octave_pclp1_2023_v1.1/check_my_octave/my_octave
==62561== Parent PID: 62560
==62561== 
==62561== 
==62561== HEAP SUMMARY:
==62561==     in use at exit: 0 bytes in 0 blocks
==62561==   total heap usage: 1,622,781 allocs, 1,622,781 frees, 12,208,808 bytes allocated
==62561== 
==62561== All heap blocks were freed -- no leaks are possible
==62561== 
==62561== For lists of detected and suppressed errors, rerun with: -s
==62561== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
