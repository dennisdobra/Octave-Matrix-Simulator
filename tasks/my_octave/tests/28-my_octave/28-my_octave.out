Unrecognized command
Unrecognized command
No matrix with the given index
